src/platforms/common/swdptap.c

```diff
@@ -70,15 +70,16 @@ static void swdptap_turnaround(const swdio_status_t dir)
 		SWDIO_MODE_FLOAT();
 	} else {
 		gpio_clear(SWCLK_PORT, SWCLK_PIN);
-		for (volatile uint32_t counter = target_clk_divider + 1; counter > 0; --counter)
-			continue;
 	}
 
+	for (volatile uint32_t counter = target_clk_divider + 1; counter > 0; --counter)
+		continue;
```

Comment:
Think this would read a little better to have a new line after the continue - personally we would move the block up two lines, but it's not a problem if you want space after the if block.. just don't want it cramping the gpio_set and its delay loop.

Our organisation pattern below was to \<change pin state\> immediately followed by \<delay loop\> then a blank new line to give space if it felt appropriate - but to have the pin state change and delay loop be the unit otherwise (two examples of this is are on lines 131 and 138).